### ISSUE TITLE
Dusk: ErrorException: ZipArchive::addFile(): Invalid or uninitialized Zip object (Alpine:edge/PHP7/Nginx+Selenium+Firefox+Laravel 5.8)

### ISSUE THREAD

**Issue by meandus** [laravel/dusk#744]
- Dusk Version: v5.9.2
- Laravel Version: v5.8.37
- PHP Version: PHP 7.3.15
- Database Driver & Version: 10.4.10-MariaDB MariaDB Server

### Description:
i tried to do an Dusk Test and i got error regarding Zip, but i'm not using download or upload zip. Only visit and assertSee()

# Error
```
docker-compose exec app php artisan dusk

PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

E                                                                   1 / 1 (100%)

Time: 5.38 seconds, Memory: 18.00 MB

There was 1 error:

**1) Tests\Browser\HomeTest::testHome
ErrorException: ZipArchive::addFile(): Invalid or uninitialized Zip object**

/var/www/vendor/php-webdriver/webdriver/lib/Firefox/FirefoxProfile.php:170
/var/www/vendor/php-webdriver/webdriver/lib/Remote/DesiredCapabilities.php:182
/var/www/vendor/php-webdriver/webdriver/lib/Remote/DesiredCapabilities.php:206
/var/www/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php:112
/var/www/tests/DuskTestCase.php:59
/var/www/vendor/laravel/dusk/src/Concerns/ProvidesBrowser.php:200
/var/www/vendor/laravel/framework/src/Illuminate/Support/helpers.php:817
/var/www/vendor/laravel/dusk/src/Concerns/ProvidesBrowser.php:201
/var/www/vendor/laravel/dusk/src/Concerns/ProvidesBrowser.php:95
/var/www/vendor/laravel/dusk/src/Concerns/ProvidesBrowser.php:65
/var/www/tests/Browser/HomeTest.php:23

ERRORS!
Tests: 1, Assertions: 0, Errors: 1.
```
# Nginx.conf
```
user                            www-data;
worker_processes                auto; # it will be determinate automatically by the number of core

error_log                       /var/log/nginx/error.log warn;
#pid                             /var/run/nginx/nginx.pid; # it permit you to use /etc/init.d/nginx reload|restart|stop|start

events {
    worker_connections          1024;
}

http {
    include                     /etc/nginx/mime.types;
    default_type                application/octet-stream;
    sendfile                    on;
    access_log                  /var/log/nginx/access.log;
    keepalive_timeout           3000;
    server {
            listen 8080;

            index index.php index.html;
            root /var/www/public;

            location / {
                try_files $uri $uri/ /index.php?$query_string;
                gzip off;
            }

            location ~ \.php$ {
                gzip off;
                try_files $uri /index.php =404;
                fastcgi_split_path_info ^(.+\.php)(/.+)$;
                fastcgi_pass php:9000;
                fastcgi_index index.php;
                fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
                include fastcgi_params;
            }
    }
}
```
# DuskTestCase.php
```
<?php

namespace Tests;

// use Facebook\WebDriver\Chrome\ChromeOptions;
use Facebook\WebDriver\Remote\DesiredCapabilities;
use Facebook\WebDriver\Remote\RemoteWebDriver;
use Laravel\Dusk\TestCase as BaseTestCase;

abstract class DuskTestCase extends BaseTestCase
{
    use CreatesApplication;

    /**
     * Prepare for Dusk test execution.
     *
     * @beforeClass
     * @return void
     */
    public static function prepare()
    {
        // static::startChromeDriver();
    }

    /**
     * Create the RemoteWebDriver instance.
     *
     * @return \Facebook\WebDriver\Remote\RemoteWebDriver
     */
    protected function driver()
    {
        return RemoteWebDriver::create(
            'http://selenium-hub:4444/wd/hub', DesiredCapabilities::firefox()
        );
    }
}
```

# HomeTest.php
```
<?php

namespace Tests\Browser;

use Tests\DuskTestCase;
use Laravel\Dusk\Browser;
use Illuminate\Foundation\Testing\DatabaseMigrations;
use App\User;

class HomeTest extends DuskTestCase
{
    /**
     * A Dusk test home.
     *
     * @return void
     */
    public function testHome()
    {
        $this->browse(function (Browser $browser) {
            $browser->loginAs(User::find(1))
                ->visit('/dashboard')
                ->assertSee('SecuriNET');
        });
    }
}
```

# docker-compose.yml:
```
  app:
    build:
      args:
        PLATEFORM: dev
      context: .
      dockerfile: Dockerfile
    image: securikeys/php-nginx-laravel:dev # alpine:edge
    container_name: app
    ports:
      - 8080:8080
    volumes:
        - ./:/var/www
    tty: true
    environment:
      SERVICE_NAME: app
      SERVICE_TAGS: dev
    depends_on:
      - db
      - redis
      - selenium-hub
    links:
      - db
      - redis
      - selenium-hub
    networks:
      - app

  selenium-hub:
    image: selenium/hub:3.141.59-zirconium
    container_name: selenium-hub
    ports:
      - "4444:4444"
    networks:
      - app
  
  firefox:
    image: selenium/node-firefox:3.141.59-zirconium
    volumes:
      - /dev/shm:/dev/shm
    depends_on:
      - selenium-hub
    environment:
      - HUB_HOST=selenium-hub
      - HUB_PORT=4444
    networks:
      - app

#Docker Networks
networks:
  app:
    driver: bridge
```
### Steps To Reproduce:
```
docker-compose exec app php artisan dusk 
```

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repo you can try one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by OndraM** [comment]
Hi, [the bug](https://github.com/php-webdriver/php-webdriver/pull/770) in FirefoxProfile which was occurring on some operating systems was fixed in php-webdriver 1.8.2, so you should update to that version (running `composer update`).
